src/tests/CTABlock.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { CTABlock } from '../components/CTABlock'
+import '@testing-library/jest-dom'
+
+test('CTABlock renders', async () => {
+  render(
+    <div data-testid="cta-block">
+      <CTABlock buttonText="Click me!" contentString={'<p>String</p>'} contentPrompt={'<p>Prompt</p>'}/>
+    </div>
+  )
+  const queryList = Array.from(screen.getByTestId('cta-block').querySelectorAll('p'))
+
+  expect(screen.getByTestId('cta-block').querySelector('button')).not.toBeNull()
+  expect(screen.getByTestId('cta-block')).toHaveTextContent('Click me!')
+  expect(queryList[0]).toHaveTextContent('String')
+  expect(queryList[1]).toHaveTextContent('Prompt')
+})
+
+test('CTABlock fires event', async () => {
+  render(
+    <div data-testid="cta-block">
+      <CTABlock buttonText="Click me!" contentString={'<p>String</p>'} contentPrompt={'<p>Prompt</p>'} fireEvent={'Event'}/>
+    </div>
+  )
+  const eventHandler = jest.fn()
+
+  document.addEventListener('Event', eventHandler)
+  fireEvent.click(screen.getByText('Click me!'))
+  expect(eventHandler).toHaveBeenCalled()
+})
+
+test('CTABlock button prompt disappear', async () => {
```

Comment:
Minor: For improved readability maybe `CTABlock button and prompt disappear on click`